### ISSUE TITLE
Add a short handle for QR-Codes

### ISSUE THREAD

**Issue by elogy** [fsi-tue/anfiheft#2]
Instead of using footnotes, I propose the new latex snippet:
```latex
\usepackage{qrcode} % should already be included
```

```latex
\newcommand{\link}[2]{
\colorbox{gray!10}{
  \parbox{\textwidth}{
  \begin{minipage}{0.2\textwidth}
    \qrcode{#1}
\end{minipage} \hfill
\begin{minipage}{0.8\textwidth}
\textbf{#2} \\ #1
\end{minipage}
}}}
```
The width of the box can be varied by adjusting `\parbox{\textwidth}` to fractions of `\textwidth`.
This will render a gray box with a QR code, a title to be displayed and the URL in clear text.
Usage:

```latex
\link{https://test.de}{Dies ist ein Test}
```

**Comment by PhictionalOne** [comment]
Your overleaf link is broken or at least overleaf reports that I am not allowed on this site.

**Comment by PhictionalOne** [comment]
Which links should be qr-**en**-coded?

Feel free to comment and select [here](https://github.com/PhictionalOne/anfiheft/commit/a38f91201055a822c5715e395c60981fa9c128ca)

**Comment by elogy** [comment]
> Your overleaf link is broken or at least overleaf reports that I am not allowed on this site.

Stupid Overleaf v2 share-link-crap.... There you go: https://www.overleaf.com/7797647179cpzvfgffwkch

**Comment by PhictionalOne** [comment]
Has been merged in #16
